pom.xml

```diff
@@ -247,7 +247,40 @@
               </goals>
             </execution>
           </executions>
-        </plugin>
+      </plugin>
+                      <plugin>
+                    <groupId>org.ec4j.maven</groupId>
+                    <artifactId>editorconfig-maven-plugin</artifactId>
+                    <version>0.1.3</version>
+                    <executions>
+                        <execution>
+                            <id>check</id>
+                            <phase>verify</phase>
+                            <goals>
+                                <goal>check</goal>
+                            </goals>
+                        </execution>
+                    </executions>
+                    <configuration>
+                        <excludes>
+                            <exclude>vocabs</exclude>
+                            <exclude>lookup-tables</exclude>
+                            <exclude>**/*.tsv</exclude>
```

Comment:
Shouldn't we include the tsv too? It has spotted a trailing whitespace which created wrong links.  Isn't that a good thing? See: https://github.com/hbz/lobid-resources/pull/1997/commits/acb15f467217d33bb2d200a17ae7cc89846d4920